src/components/partner/partner.repository.ts

```diff
@@ -140,6 +141,27 @@ export class PartnerRepository extends DtoRepository<
           relation('out', '', 'languageOfWiderCommunication', ACTIVE),
           node('languageOfWiderCommunication', 'Language'),
         ])
+        .subQuery('node', (sub) =>
+          sub
+            .match([
+              node('node'),
+              relation('out', '', 'languagesOfConsulting', ACTIVE),
+              node('languagesOfConsulting', 'Language'),
+            ])
+            .return(
+              'collect(languagesOfConsulting.id) as languagesOfConsultingIds',
+            ),
+        )
+        .with([
```

Comment:
This is the reason the test suite is failing. Since you started this, the query above was modified to add an additional variable that is not included in this list.
I still stand behind the fact that this clause should not be needed.
If you are having trouble without, perhaps moving above the `matchProps` clause.